routes/playground.janet

```diff
@@ -0,0 +1,40 @@
+(import joy :prefix "")
+(import ../helpers :prefix "")
+
+(defn playground [request code]
+  #(def account (current-account request))
+  (def html
+  [:vstack {:spacing "l"}
+    [:script {:src "/ace/ace.js"}]
+    [:hstack {:spacing "s"}
+        [:button {:id "run" :title="ctrl-enter"} "Run"]
+        [:button {:id "format"} "Format"]
+        [:spacer]]
+      [:div {:id "code" :class "hljs" :style "height:60vh;"} (raw code)]
+      [:pre {:id "output" :style "overflow:auto;"}]
+      [:div {:id "hiddencode" :style "display:none;"}]
+      [:div {:id "sporkformat" :style "display:none;"}
+       (string (slurp "public/playground/fmt.janet"))]
+    [:script {:type "text/javascript" :src "/playground/playground.js" :async "false"}]
+    [:script {:type "text/javascript" :src "/playground/janet.js" :async "false"}]
+    [:script {:type "text/javascript" :src "/playground/jdocs_playground.js" :async "false"}] 
+    ])
```

Comment:
It's not a big deal but you could use something like [parinfer](https://shaunlebron.github.io/parinfer/) to align these parens